tests/drivers/entropy/api/entropy_psa_crypto.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_TFM_PARTITION_CRYPTO=y
+CONFIG_TFM_CRYPTO_RNG_MODULE_ENABLED=y
+CONFIG_ENTROPY_PSA_CRYPTO_RNG=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_HARDWARE_DEVICE_CS_GENERATOR=y
```

Comment:
You do not need to set
```
CONFIG_ZTEST=y
CONFIG_ZTEST_NEW_API=y
CONFIG_ENTROPY_GENERATOR=y
```
here because these symbols are already in `prj.conf`